lib/modules/platform/util.ts

```diff
@@ -15,3 +16,12 @@ export function getNewBranchName(branchName?: string): string | undefined {
   }
   return branchName;
 }
+
+export function getDefaultPlatformPrOptions(): PlatformPrOptions {
+  return {
+    azureAutoApprove: false,
+    azureWorkItemId: 0,
+    bbUseDefaultReviewers: false,
+    gitLabIgnoreApprovals: 0,
+  };
+}
```

Comment:
@viceice Do you think this is a good idea?

Reply:
no, should be handled inside getplatformoptions